aiida/repository/backend/sandbox.py

```diff
@@ -17,6 +18,10 @@ class SandboxRepositoryBackend(AbstractRepositoryBackend):
     def __init__(self):
         self._sandbox = None
 
+    def __str__(self) -> str:
+        """Return the string representation of this repository."""
+        return f'SandboxRepository: {self._sandbox.abspath}'
```

Comment:
The UUID is not being returned here, might be nice? Also whether or not it's initialised?

Reply:
Sandbox doesn't have a UUID and always returns `None`. It doesn't make sense for a sandbox tbh

Reply:
Ah right. Yeah. There were so many layers here I kind of lost track of which string is represented where 😅